### PR TITLE
Fix thrown error when calling Patch::Sdg

### PR DESCRIPTION
**Pull request by biglovisa**
### What

Requiring the SDG model when instantiating the gem. 

### Why

Without this LOC, we get a NameError whenever calling `Patch::Sdg` since it was not required. Tests have been added to make sure that all models present in `lib/patch_ruby/models` are required. 

<details>
<summary>Test run of before/after the fix</summary>

Screen Shot 2021-03-02 at 08 20 47

</details> 

<details>
<summary>Screenshot of triggering the error in version 1.5.0</summary>

Screen Shot 2021-03-02 at 08 39 47

</details>

<details>
<summary>Screenshot of triggering the error in version 1.5.1 </summary>
Screen Shot 2021-03-02 at 08 26 52

</details>

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [x] Have you built the gem locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version?
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: lib/patch_ruby.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@
 require 'patch_ruby/models/project'
 require 'patch_ruby/models/project_list_response'
 require 'patch_ruby/models/project_response'
+require 'patch_ruby/models/sdg'
```
Comment (biglovisa) [inline lib/patch_ruby.rb:39]: The model wasn't required, which caused the error.

File: spec/patch_ruby_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  context 'Models' do
+    it 'defines all models' do
+      constants.each do |constant|
+        expect { Patch.const_get(constant) }.not_to raise_error
+      end
+    end
+  end
+
+  def constants
+    # Given a file path return the constant of that path, for example:
```
Comment (biglovisa) [inline spec/patch_ruby_spec.rb:11]: All recommendations on how to achieve this in a smoother way is welcome. I'm sure there's a more straightforward approach to this that I can't think of right now.

File: Gemfile.lock [hunk 3]
```diff
@@ -26,7 +26,7 @@ GEM
     ffi (1.14.2)
     i18n (1.8.7)
       concurrent-ruby (~> 1.0)
-    json (2.5.1)
+    json (2.4.1)
```
Comment (pcothenet) [inline Gemfile.lock:29]: Why this change?

File: spec/patch_ruby_spec.rb (hunk 2 above)
Comment (pcothenet) [reply spec/patch_ruby_spec.rb:11]: That looks pretty OK for now! Let's do a survey of other SDKs a bit later for best practices?

**Review by pcothenet — APPROVED** [review]

File: Gemfile.lock (hunk 3 above)
Comment (biglovisa) [reply Gemfile.lock]: no reason - reverted this! Thanks for the callout
